test/unit/edition_test.rb

```diff
@@ -936,4 +936,12 @@ class EditionTest < ActiveSupport::TestCase
     assert edition.has_secondary_sectors?
     assert edition.has_legacy_tags?
   end
+
+  test 'should raise an exception when attempting to modify an edition of a locked document' do
```

Comment:
The division between this test file and `test/unit/edition/validation_test.rb` is a bit muddled to say the least, but on further investigation I think this file is more about testing methods on Edition rather than validation, so I think we should move this test into the validation test rile and change the description to reflect that an exception is raised when attempting to create a new edition for a locked document.